py/circuitpy_mpconfig.mk

```diff
@@ -74,8 +74,13 @@ CIRCUITPY_AUDIOIO = $(CIRCUITPY_FULL_BUILD)
 endif
 CFLAGS += -DCIRCUITPY_AUDIOIO=$(CIRCUITPY_AUDIOIO)
 
+ifndef CIRCUITPY_AUDIOPWMIO
+CIRCUITPY_AUDIOPWMIO = 0
+endif
+CFLAGS += -DCIRCUITPY_AUDIOPWMIO=$(CIRCUITPY_AUDIOPWMIO)
+
 ifndef CIRCUITPY_AUDIOCORE
-CIRCUITPY_AUDIOCORE = $(CIRCUITPY_AUDIOIO)
+CIRCUITPY_AUDIOCORE = $(if $(filter 1, $(CIRCUITPY_AUDIOPWMIO) $(CIRCUITPY_AUDIOBUSIO) $(CIRCUITY_AUDIOIO)),1,0)
```

Comment:
Is it possible this isn't working when `CIRCUITPY_AUDIOIO = $(CIRCUITPY_FULL_BUILD)` is late bound?

Reply:
The explanation must be something like that.  I'll dial back the cleverness here.